openedx_tagging/core/tagging/import_export/exceptions.py

```diff
@@ -0,0 +1,62 @@
+from django.utils.translation import gettext_lazy as _
+
+class ImportError(Exception):
```

Comment:
[ImportError](https://docs.python.org/3/library/exceptions.html#ImportError) is a built-in exception.. can this be TagImportError instead?

Reply:
Fixed here https://github.com/openedx/openedx-learning/pull/64/commits/b056500d9fdaf1b23217d5fa45a7e96bfb621286